sections/image-banner.liquid

```diff
@@ -154,11 +173,31 @@
                   {% else %}
                     href="{{ block.settings.button_link_2 }}"
                   {% endif %}
-                  class="button{% if block.settings.button_style_secondary_2 %} button--secondary{% else %} button--primary{% endif %}"
+                  class="button banner-button banner-button-second{% if block.settings.button_style_secondary_2 %} button--secondary{% else %} button--primary{% endif %}"
```

Comment:
instead of adding your custom classes for every element. Give custom class to parent and add custom css using parent class